src/supertux/menu/game_menu.cpp

```diff
@@ -37,6 +38,17 @@ GameMenu::GameMenu()
   add_submenu(_("Options"), MenuStorage::INGAME_OPTIONS_MENU);
   add_hl();
   add_entry(MNID_ABORTLEVEL, _("Abort Level"));
+
+  reset_callback = [] {
```

Comment:
Please move these initializations into the initialization list, and thereby make Travis happy.